test_googleMapsApiModule.py

```diff
@@ -5,13 +5,15 @@
 
 
 class TestClass:
+    @pytest.mark.skipif(googleMapsApiModule.gmaps is None, reason="No API key provided")
```

Comment:
Workaround for Travis CI API key errors.
Discussion [here](https://github.com/vishakha-lall/MapBot/pull/103#issuecomment-607411562).